rfcs/text/0000-data-source-categorization-fields.md

```diff
@@ -0,0 +1,73 @@
+# 0000: Data Source Categorization Fields
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **August 26 2020** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+Elastic currently supports ingestion of data from 180+ sources, and growing. However, we do not have a coherent way to categorise these sources. This has resulted in a disconnect in how we categorize these sources from the Elastic website, in-product experiences and ECS.
+
+The fieldset we use to describe the data source is up for discussion, data_stream.category is a possibility. Here are proposed allowed values:
+
+- apm
```

Comment:
Small thing.  I suggest we standardize on the capitalization and naming.  For example we have an event.category of "iam" but a proposed data_stream.category of "Identity and access management".  Also we have an example of "ids" for observer.type and a proposed data_stream.category of "IDS".

Reply:
+1